build-docs.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+branch=$(git branch --show-current)
+
+make k8s
+make linux
+make container
+make windows
+make macos
+
+
+sudo mkdir -p /var/www/docs/minio/kubernetes/upstream
+sudo mkdir -p /var/www/docs/minio/linux
+sudo mkdir -p /var/www/docs/minio/windows
+sudo mkdir -p /var/www/docs/minio/macos
+sudo mkdir -p /var/www/docs/minio/container
```

Comment:
Request we clean the directory contents, unless we have some other spot where we do that cleanup.

Sphinx has a behavior where if FILE.rst includes CONTENT.rst, and only CONTENT.rst changes, it does not update FILE.rst on build.

Only cleaning the contents of the build directory fixes this.

Reply:
Maybe adding --remove-destination would do the trick here as it would re-write all files on collision

Reply:
or -u to update, since source will always be newer by virtue of a newer build.